src/components/ebay-icon/symbols/alert/ds6.marko

```diff
@@ -1,4 +1,4 @@
-<symbol id="icon-alert">
+<symbol id="icon-alert" viewBox="0 0 16 16">
```

Comment:
I see the change for all the icons, though, not just the missing one.

Reply:
Okay, ignore me. I see now. These SVG files are for when they've used an old DS4 (now-reoved in DS6) icon, correct?

Reply:
Yes, that or if there is a DS6 icon which does not exist in DS4